datafusion/core/benches/parquet_statistic.rs

```diff
@@ -70,7 +84,14 @@ fn create_parquet_file(dtype: TestTypes, row_groups: usize) -> NamedTempFile {
         )])),
     };
 
-    let props = WriterProperties::builder().build();
+    let mut props = WriterProperties::builder();
```

Comment:
🤔  while looking at this code, It seems like the benchmark for row groups doesn't actually make multiple row groups (it instead it encodes `row_goups` batches instead)

We should probably also set https://docs.rs/parquet/latest/parquet/file/properties/struct.WriterProperties.html#method.max_row_group_size as well 🤔

Reply:
i fixed this. however, it should not make any difference since we have it set at `100` but only have 5 rows in our batch